requirements.txt

```
@@ -1,30 +1,30 @@
 -e .[asyncio,trio,http2,socks]
 
 # Docs
-mkdocs==1.6.0
-mkdocs-autorefs==1.0.1
-mkdocs-material==9.5.27
+mkdocs==1.6.1
+mkdocs-autorefs==1.4.2
```

Later versions seem to break Python 3.8.
```suggestion
mkdocs-autorefs==1.2.0
```